### PR TITLE
ISPN-11469 Embedded tutorials fixed

### PR DESCRIPTION
**Pull request by karesti**
https://issues.jboss.org/browse/ISPN-11469

### CODE REVIEW

File: distributed/src/main/java/org/infinispan/tutorial/simple/distributed/InfinispanDistributed.java [hunk 1]
```diff
@@ -14,13 +15,16 @@
...
+      //Create cache configuration
       ConfigurationBuilder builder = new ConfigurationBuilder();
       builder.clustering().cacheMode(CacheMode.DIST_SYNC);
-      // Initialize the cache manager
-      DefaultCacheManager cacheManager = new DefaultCacheManager(global.build(), builder.build());
-      // Obtain the default cache
-      Cache<String, String> cache = cacheManager.getCache();
+      // Obtain a cache
+      Cache<String, String> cache = cacheManager.administration()
+              .withFlags(CacheContainerAdmin.AdminFlag.VOLATILE)
```
Comment (danberindei) [inline distributed/src/main/java/org/infinispan/tutorial/simple/distributed/InfinispanDistributed.java]: The indent here is 2*4 spaces, but in most other places it's 3 spaces. 

I suggest keeping the indent 3 spaces, and for fluent methods I would prefer aligning them on the `.`, e.g.

```java
      Cache<String, String> cache =
            cacheManager.administration().withFlags(CacheContainerAdmin.AdminFlag.VOLATILE)
                        .getOrCreateCache("cache", builder.build());
```

 (In IntelliJ settings, Code Style, Wrapping and Braces, Chained method calls, Align when multiline.)

File: streams/src/main/java/org/infinispan/tutorial/simple/streams/InfinispanStreams.java [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
-
 import org.infinispan.Cache;
 import org.infinispan.configuration.cache.ConfigurationBuilder;
 import org.infinispan.manager.DefaultCacheManager;
-import org.infinispan.stream.CacheCollectors;
+
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
```
Comment (danberindei) [inline streams/src/main/java/org/infinispan/tutorial/simple/streams/InfinispanStreams.java]: The Infinispan code style is to import `javax` and `java` packages first, let's keep it the same in the tutorials.

Comment (karesti) [reply streams/src/main/java/org/infinispan/tutorial/simple/streams/InfinispanStreams.java]: I don't understand what happened here with formatters

**Comment by karesti** [comment]
@danberindei is it ok now ?

File: streams/src/main/java/org/infinispan/tutorial/simple/streams/InfinispanStreams.java [hunk 3]
```diff
@@ -24,9 +20,8 @@ public static void main(String[] args) {
       int range = 10;
       IntStream.range(0, range).boxed().forEach(i -> cache.put(i + "-key", i + "-value"));
       // Map and reduce the keys
-      int result = cache.keySet().stream()
-         .map(e -> Integer.valueOf(e.substring(0, e.indexOf("-"))))
-              .collect(() -> Collectors.summingInt(i -> i.intValue()));
+      int result = cache.keySet().stream().map(e -> Integer.valueOf(e.substring(0, e.indexOf("-"))))
```
Comment (danberindei) [inline streams/src/main/java/org/infinispan/tutorial/simple/streams/InfinispanStreams.java]: I would keep the `map` on a new line. As I mentioned before, I prefer aligning the chained method calls on the `.` on the first line, but it's not required if you don't like it :)

E.g.
```java
int result = cache.keySet().stream()
                  .map(e -> Integer.valueOf(e.substring(0, e.indexOf("-"))))
                  .collect(() -> Collectors.summingInt(i -> i.intValue()));
```

**Comment by karesti** [comment]
@danberindei should be good now

**Comment by danberindei** [comment]
Merged, thanks Katia!

**Pull request merged**